app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -389,11 +389,9 @@
   <dimen name="solution_summary_margin_top_linear_layout">0dp</dimen>
   <dimen name="solution_summary_margin_bottom_linear_layout">32dp</dimen>
   <dimen name="solution_summary_margin_top_solution_summary_label">16dp</dimen>
-
   <!-- SectionTitleFragment -->
   <dimen name="section_title_divider_view_layout_margin_top">36dp</dimen>
   <dimen name="section_title_text_view_layout_margin_start">96dp</dimen>
   <dimen name="section_title_text_view_layout_margin_top">32dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">96dp</dimen>
-
 </resources>
```

Comment:
Can you reintroduce the spaces as they were earlier.

Reply:
Okk,I will add these spaces.